src/server/routes/api.ts

```diff
@@ -121,6 +124,17 @@ connection.once('open', () => {
             g.study = g.study;
             g.tissue = g.tissue;
 
+            // TODO: this map may not be needed
```

Comment:
What will determine it?

Reply:
Yes, will update or remove the comment once we get the newest data file. Thanks for reminding me! :)